pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.2.3</Version>
+    <Version>5.2.4-beta</Version>
```

Comment:
@yizhang82 can correct me if I'm wrong on any of the details, but we are planning on publishing the metapackage/Microsoft.Net.Native.Compiler package at the time of Dev15 RC. The in-box template will specify version 5.2.2 of the metapackage, so developers who want to try our NuGet toolchain experience will have to manually update their metapackage.